types/sharedb/lib/sharedb.d.ts

```diff
@@ -133,6 +152,33 @@ export class Query extends EventEmitter {
     destroy(): void;
 }
 
+export type ReceivePresence<T> = (id: string, value: T) => void;
```

Comment:
I wasn't sure what this was at first, before noticing it's a function. A name like `ReceivePresenceListener` (or `ReceivePresenceCallback` like you suggested verbally) could make this clearer